src/javascript/app/pages/cashier/dp2p.js

```diff
@@ -117,6 +169,9 @@ const DP2P = (() => {
                 local_currency_config: Client.get('local_currency_config'),
                 residence            : Client.get('residence'),
             },
+            custom_strings: {
+                email_domain: 'binary.com',
```

Comment:
didn't you make this change in your other PR?

Reply:
removing it 🙏